lib/src/main/java/org/esbtools/lightbluenotificationhook/NotificationHook.java

```diff
@@ -219,7 +211,29 @@ private NotificationEntity makeNotificationEntityWithIncludedFields(HookDoc hook
                 data.add(new NotificationEntity.PathAndValue(pathString, valueString));
             }
         }
-        
+        List<String> changedPaths=new ArrayList<>();
+        List<NotificationEntity.PathAndValue> removedPaths=new ArrayList<>();
+        for(DocComparator.Delta<JsonNode> delta:diff.getDelta()) {
+            if( (delta instanceof DocComparator.Move && arrayOrderSignificant) ||
+                !(delta instanceof DocComparator.Move) ) {
+
+                if(delta instanceof DocComparator.Removal) {
+                    removedPaths.add(new NotificationEntity.
+                                     PathAndValue(delta.getField().toString(),
+                                                  ((DocComparator.Removal)delta).
+                                                  getRemovedNode().toString()));
+                } else {
+                    if(delta instanceof DocComparator.Move) {
+                        // Add the new path to the changedPaths
+                        changedPaths.add(delta.getField2().toString());
+                    } else {
+                        changedPaths.add(delta.getField().toString());
+                    }
+                }
+            }
+        }
+        notificationEntity.setChangedPaths(changedPaths);
+        notificationEntity.setRemovedPaths(removedPaths);
```

Comment:
Since the type of this is `List<PathAndValue>` like `entityData`, I'd rather call this `removedEntityData` or something of the sort to be consistent. Otherwise it appears this should be a `List<String>` since that's what `changedPaths` is and they share the same noun.